counterr/counters.py

```diff
@@ -89,15 +96,27 @@ def mapQ_stats_aligned_readsegment(reads, verbose):
     stds_out = []
     lens_out = []
     for name in reads.keys():
-        for read in reads[name]: # Iterate through the reads
+        for read in reads[name]:  # Iterate through the reads
             # Inside aligned region
             Q_in = read.query_alignment_qualities
-            means_in.append(np.mean(Q_in))
-            stds_in.append(np.std(Q_in))
+            # Q_in is None implies that there are no alignment qualities, so we
+            # append dummy values
+            if Q_in is None:
+                means_in.append(0)
+                stds_in.append(0)
+            else:
+                means_in.append(np.mean(Q_in))
+                stds_in.append(np.std(Q_in))
             lens_in.append(read.query_alignment_length)
             # Outside aligned region
             Q = read.query_qualities
-            Q_out = np.concatenate([Q[:read.query_alignment_start], Q[read.query_alignment_end:]])
+            # There may not be qualities for the read, so creating an empty
```

Comment:
Same thing here. When read qualities aren't present, we run into the same issue.